plugins/sql/guest-js/index.ts

```diff
@@ -42,12 +42,31 @@ export default class Database {
    *
    * @example
    * ```ts
+   * // Basic connection to a SQLite database
    * const db = await Database.load("sqlite:test.db");
+   *
+   * // Connecting with an encryption key for added security
+   * const db = await Database.load("sqlite:encrypted.db", {
+   *   sqlite: { pragmas: { "key": "encryption_key" } }
+   * });
+   *
+   * // Connecting with specific pragmas for configuration
+   * const db = await Database.load("sqlite:test.db", {
+   *   sqlite: { pragmas: { "journal_mode": "WAL", "foreign_keys": "ON" } }
+   * });
    * ```
    */
-  static async load(path: string): Promise<Database> {
+  static async load(
+    path: string,
+    options?: {
+      sqlite?: {
+        pragmas?: Record<string, string>
```

Comment:
Since you are already adding `options`. Could you add support for extensions, to fix #1705
```
extensions?: Record<string, Option<string>>  // extension, entry_point
```